plugins/source/alicloud/client/spec.go

```diff
@@ -49,3 +54,38 @@ func (s *Spec) Validate() error {
 	}
 	return nil
 }
+
+func (Spec) JSONSchemaExtend(sc *jsonschema.Schema) {
+	one := uint64(1)
+
+	accounts := *sc.Properties.Value("accounts").OneOf[0]
+	accounts.MinItems = &one
+	sc.Properties.Set("accounts", &accounts)
+
+	sc.Required = append(sc.Required, "accounts")
+}
+
+func (AccountSpec) JSONSchemaExtend(sc *jsonschema.Schema) {
+	one := uint64(1)
+
+	name := *sc.Properties.Value("name")
+	name.MinLength = &one
```

Comment:
use annotation instead
```go
type AccountSpec struct {
	Name      string   `json:"name,omitempty" jsonschema:"required,minLength=1"`
	Regions   []string `json:"regions,omitempty" jsonschema:"required,minItems=1"`
	AccessKey string   `json:"access_key,omitempty" jsonschema:"required,minLength=1"`
	SecretKey string   `json:"secret_key,omitempty" jsonschema:"required,minLength=1"`
}
```

Reply:
Addressed in https://github.com/cloudquery/cloudquery/pull/16355/commits/baa62595a3f268025cc062b4d5900fe069e4e3bd